src/assets/scss/material.orcid.overwrites.scss/button.scss

```diff
@@ -21,10 +21,13 @@ button.mat-button-wrap-text-with-icon {
   border: solid 1px rgba(0, 0, 0, 0.12);
   margin-top: 12px;
 
-  mat-icon {
+  a {
     position: absolute;
     left: 5%;
```

Comment:
Let's not use percentage for these margins, this will make the button look unaligned on some screen sizes. 

![image](https://user-images.githubusercontent.com/2119626/82618785-43e70c00-9b91-11ea-8477-72176c34879b.png)

Also for Arabic, we might rather have these icons on the right, some docs on material about this https://material.io/design/usability/bidirectionality.html#mirroring-elements

Reply:
Done